code/src/reducers/todos.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const todos = createSlice({
+  name: 'todos',
+  initialState: {
+    items: [
+    { id: 1, category: 'Study', text: 'Watch video on actions & reducers', complete: true, dueDate: Date.now() },
```

Comment:
Since you have so much information here then I would split these up: 
        id: 1,
        category: "Study",
        text: "Watch video on actions & reducers",
        complete: true,
        dueDate: Date.now(),

Reply:
Yes, could definitely be done. Will probably remove these from the initial state also when I get around to update it :)